components/clp-package-utils/clp_package_utils/scripts/start_clp.py

```diff
@@ -538,13 +555,22 @@ def start_query_scheduler(
     mounts: CLPDockerMounts,
 ):
     module_name = "job_orchestration.scheduler.query.query_scheduler"
+    query_scheduler_mount = None
```

Comment:
once you use vars like QUERY_SCHEDULER_COMPONENT_NAME as 2nd argument, maybe you can then refactor this logic into `generic_start_scheduler`.